src/transaction-runner.coffee

```diff
@@ -9,13 +9,18 @@ clone = require 'clone'
 caseless = require 'caseless'
 {Pitboss} = require 'pitboss-ng'
 
-flattenHeaders = require './flatten-headers'
 addHooks = require './add-hooks'
 sortTransactions = require './sort-transactions'
 packageData = require './../package.json'
 logger = require './logger'
 
 
+headersArrayToObject = (arr) ->
+  obj = {}
+  obj[name] = value for {name, value} in arr
+  return obj
```

Comment:
Transforming header array -> header object is probably not what you want - headers with the same name get overwritten in the process and only last one will remain.

Reply:
I should have been more specific - this just enables _future implementation_ of https://github.com/apiaryio/dredd/issues/356 😄